lib/src/markdown_processor.dart

```diff
@@ -125,6 +119,21 @@ const validHtmlTags = const [
 final nonHTMLRegexp =
     new RegExp("</?(?!(${validHtmlTags.join("|")})[> ])\\w+[> ]");
 
+// Type parameters and other things to ignore at the end of doc references.
+final trailingIgnoreStuff = new RegExp(r'(<.*>|\(.*\))$');
```

Comment:
I prefer having the type for LHS (`final RegExp trailingIgnoreStuff`), but I believe that this is not a mandated part of the dart style guide.